src/fondant/pipeline/compiler.py

```diff
@@ -45,6 +47,37 @@ def log_unused_configurations(self, **kwargs):
                     f" for runner `{self.__class__.__name__}`.",
                 )
 
+    @staticmethod
+    def _build_entrypoint(image: Image) -> t.List[str]:
```

Comment:
Don't really know how to test this without either replicating the logic or comparing against an unreadable string, so for now this is only tested via the integration test.

Open for ideas.

Reply:
IMO comparing to a hard to read string is fine

Reply:
Not sure how valuable it is though, because I would just copy paste the generated string at this moment. It would be the old compiler tests all over again :)